src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/resource/CsipRecordsController.kt

```diff
@@ -140,7 +139,7 @@ class CsipRecordsController(val csipRecordService: CsipRecordService, val prison
       ),
     ],
   )
-  @PreAuthorize("hasAnyRole('$ROLE_CSIP_UI', '$ROLE_NOMIS')")
+  @PreAuthorize("hasAnyRole('$ROLE_CSIP_UI')")
```

Comment:
Sync will still need to retrieve the CSIP record for the DPS -> NOMIS sync direction